themes/default/css/_besocial/index_besocial.css

```diff
@@ -1663,19 +1663,19 @@ a.help .icon {
 }
 
 .i-comment::before {
-	content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' stroke='black' stroke-width='6em' fill='%231ab31a' viewBox='0 0 3584 3584'%3E%3Cpath d='M3584 1792q0 348-240 643t-652 466-900 171q-140 0-290-16-396 350-920 484-98 28-228 44-34 4-61-18t-35-58v-2q-6-8-1-24t4-20 9-19l12-18t14-17 16-18q14-16 62-69t69-76 62-79 65-102 54-118 52-152q-314-178-495-440t-181-562q0-260 142-497t382-409 572-273 696-101q488 0 900 171t652 466 240 643z'/%3E%3C/svg%3E");
+	content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' stroke='black' stroke-width='6em' fill='#43a8da' viewBox='0 0 3584 3584'%3E%3Cpath d='M3584 1792q0 348-240 643t-652 466-900 171q-140 0-290-16-396 350-920 484-98 28-228 44-34 4-61-18t-35-58v-2q-6-8-1-24t4-20 9-19l12-18t14-17 16-18q14-16 62-69t69-76 62-79 65-102 54-118 52-152q-314-178-495-440t-181-562q0-260 142-497t382-409 572-273 696-101q488 0 900 171t652 466 240 643z'/%3E%3C/svg%3E");
```

Comment:
You don't have an IE9 VM? :P

https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/linux/

Anyway, if it's live on the site atm then the icons seem to be working
okay, but of course the linear-gradient without specifying a solid bg color
first seperately is not.

So it says:

background: #F4F4F4 linear-gradient(to bottom, #FEFEFE, #EEEEEE);

should be:

background: #F4F4F4;
background: linear-gradient(to bottom, #FEFEFE, #EEEEEE);

On Tue, Apr 26, 2016 at 8:01 PM, Vekseid <notifications@github.com> wrote:

> In themes/default/css/_besocial/index_besocial.css
> <https://github.com/elkarte/Elkarte/pull/2514#discussion_r61134324>:
>
> > @@ -1663,19 +1663,19 @@ a.help .icon {
> >  }
> >
> >  .i-comment::before {
> > -	content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' stroke='black' stroke-width='6em' fill='%231ab31a' viewBox='0 0 3584 3584'%3E%3Cpath d='M3584 1792q0 348-240 643t-652 466-900 171q-140 0-290-16-396 350-920 484-98 28-228 44-34 4-61-18t-35-58v-2q-6-8-1-24t4-20 9-19l12-18t14-17 16-18q14-16 62-69t69-76 62-79 65-102 54-118 52-152q-314-178-495-440t-181-562q0-260 142-497t382-409 572-273 696-101q488 0 900 171t652 466 240 643z'/%3E%3C/svg%3E");
> > +	content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' stroke='black' stroke-width='6em' fill='#43a8da' viewBox='0 0 3584 3584'%3E%3Cpath d='M3584 1792q0 348-240 643t-652 466-900 171q-140 0-290-16-396 350-920 484-98 28-228 44-34 4-61-18t-35-58v-2q-6-8-1-24t4-20 9-19l12-18t14-17 16-18q14-16 62-69t69-76 62-79 65-102 54-118 52-152q-314-178-495-440t-181-562q0-260 142-497t382-409 572-273 696-101q488 0 900 171t652 466 240 643z'/%3E%3C/svg%3E");
>
> Do unencoded hashmarks work in all browsers here? Namely IE9?
>
> Would simplify things, it has gotten annoying.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/elkarte/Elkarte/pull/2514/files/1fbfb9ce520a88a92a900e3fb07436c8cf99a9b9..40c8f04a1b395a7b0f4b7f0c3532e8a1193adfec#r61134324>
>